sankaristoorit-backend/app.js

```diff
@@ -16,7 +16,7 @@ mongoose.set('useFindAndModify', false)
 mongoose.connect(conf.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true })
   .then(() => {
     console.log('connected to MongoDB')
-    if (process.env.NODE_ENV === 'test') {
+    if (process.env.NODE_ENV === 'test' && process.env.JEST_WORKER_ID === undefined) {
```

Comment:
When jest is running, JEST_WORKER_ID is set. When running Cypress tests, NODE_ENV==='test' and JEST_WORKER_ID===undefined, so testInitializer will only run then. Running testInitializer with jest created problems, and actually it is not needed for the jest tests.